packages/dos-donts-block/src/DoDontItem.tsx

```diff
@@ -154,15 +154,19 @@ export const DoDontItem = React.forwardRef<HTMLDivElement, DoDontItemProps>(
         }, [designTokens, body, id, editing, onChangeItem]);
 
         return (
-            <div ref={ref} style={{ ...transformStyle, ...(isMenuOpen ? { zIndex: 2 } : {}) }}>
+            <div
+                ref={ref}
+                className="hover:!tw-z-[2]"
```

Comment:
Could you also pass the z index in the className and remove it from style? like: 
`className={joinClassNames(["hover:!tw-z-[2]", isMenuOpen && "tw-z-[2]"])} style={transformStyle}`

Reply:
& Why exactly z-[2] and not z-10?

Reply:
Not possible unfortunately, because when dragging, {...transformStyle} adds a zIndex:1, so I need to override that with zIndex:2 if I want that item to be on top.